.codecov.yml

```diff
@@ -0,0 +1,16 @@
+coverage:
+  status:
+    project:
+      default:
+        # this will change as tests are added
+        target: 50%
+        threshold: 5%
```

Comment:
What does the threshold do?

Reply:
Threshold is the amount the coverage is allowed to drop on a new PR to pass.

For example, our current coverage is 60%. If you added some new stuff and didn't test it all and it dropped to 58%, that would be fine. But if it dropped to 54% that would not be fine.

Ideally, we would set our threshold to 1 or 2%. 5% percent is for while our testing is still in early stages.

Reply:
Nicely worded explanation, clear format with just a hint of bread and possibly a dash of cinnamon. Thank you